src/config.c

```diff
@@ -514,12 +540,20 @@ void loadServerConfigFromString(char *config) {
         } else if (!strcasecmp(argv[0],"loadmodule") && argc >= 2) {
             queueLoadModule(argv[1],&argv[2],argc-2);
         } else if (strchr(argv[0], '.')) {
-            if (argc != 2) {
+            if (argc < 2) {
                 err = "Module config specified without value";
                 goto loaderr;
             }
             sds name = sdsdup(argv[0]);
-            if (!dictReplace(server.module_configs_queue, name, sdsdup(argv[1]))) sdsfree(name);
+            sds val;
+            if (argc == 2) {
+                val = sdsdup(argv[1]);
+            } else {
+                val = sdscatfmt(sdsempty(), "%S", argv[1]);
+                for (int i=2; i<argc; i++)
+                    val = sdscatfmt(val, " %S", argv[i]);
+            }
```

Comment:
```suggestion
```suggestion
            sds val = sdsdup(argv[1]);
            for (int i=2; i<argc; i++)
                val = sdscatfmt(val, " %S", argv[i]);
```
```